Tweak a template of newgem

koic opened the pull request:
if value of `config[:coc]` is not `true` when executing `bundle gem <gemname>`, redundant blank lines are created in README.md.

This patch will not create these redundant blank lines.

```diff
 ## Contributing

 Bug reports and pull requests are welcome on GitHub at https://github.com/[USERNAME]/foo.

-
 ## License

 The gem is available as open source under the terms of the [MIT License]
 (http://opensource.org/licenses/MIT).
-
```

Related PR: #5523

colby-swandale commented:
Thanks for opening a PR and helping make Bundler better!

I'm unable to reproduce the problems with the paragraph spacing that your PR is addressing. If i generate a new gem with the code of conduct set to `false`, the header for `### License ###` is not being spaced correctly.

Here is a `README` from you're changes:
```
## Contributing

Bug reports and pull requests are welcome on GitHub at https://github.com/[USERNAME]/testapp. This project is intended to be a safe, welcoming space for collaboration, and contributors are expected to adhere to the [Contributor Covenant](http://contributor-covenant.org) code of conduct.
## License

The gem is available as open source under the terms of the [MIT License](http://opensource.org/licenses/MIT).

## Code of Conduct

Everyone interacting in the Testapp project’s codebases, issue trackers, chat rooms and mailing lists is expected to follow the [code of conduct](https://github.com/[USERNAME]/testapp/blob/master/CODE_OF_CONDUCT.md).
```

Also if i generate a new gem on `master` i'm also not seeing any issues with the spacing. Can you share a way to reproduce this?

koic commented:
Thanks for your quick reply and confirmation.

It was my misunderstanding about the blank line above `## License` :bow:

This PR merely omit a blank line of the end of README.md created when config `config[:coc]` is not `true`.

```diff
 ## Contributing

 Bug reports and pull requests are welcome on GitHub at https://github.com/[USERNAME]/foo.

 ## License

 The gem is available as open source under the terms of the [MIT License]
 (http://opensource.org/licenses/MIT).
-
```

I fixed it and squashed.

colby-swandale commented:
no worries and thank you! 🎉 

@bundlerbot r+
